objcxx_eh.cc

```diff
@@ -15,7 +15,7 @@ namespace __cxxabiv1
 
 using __cxxabiv1::__class_type_info;
 
-#ifndef USE_STD_EXCEPTION_TYPES
+#ifndef TYPE_INFO_IS_EXPOSED
```

Comment:
![image](https://user-images.githubusercontent.com/15292574/38554430-491d8a4a-3ccb-11e8-9ea8-9821e2d04557.png)
I can remove it from this merge request, since it probably deserves an issue on its own.
No test seems to fail even after removing it though.